notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb

```diff
@@ -338,23 +332,12 @@
    "execution_count": 11,
    "id": "dd7628bf",
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "MobileNetV3_large_x1_0.xml already exists.\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "from openvino.tools import mo\n",
-    "from openvino.runtime import serialize\n",
-    "\n",
     "model_xml = Path(MODEL_NAME).with_suffix('.xml')\n",
     "if not model_xml.exists():\n",
-    "    ov_model = mo.convert_model(\"model/MobileNetV3_large_x1_0_infer/inference.pdmodel\")\n",
-    "    serialize(ov_model, str(model_xml))\n",
+    "    ov_model = ov.convert_model(\"model/MobileNetV3_large_x1_0_infer/inference.pdmodel\")\n",
+    "    ov.save_model(ov_model, str(model_xml))\n",
```

Comment:
String wrapping for path can be removed everywhere

Reply:
It is not a part of the task